CHANGELOG.md

```diff
@@ -1,14 +1,12 @@
 # Beadledom Changelog
 
-## 3.1 - TBD
+## 3.1 - 12 September 2018
 
 ### Additions
 * Add offset based pagination with beadledom-pagination.
 
-### Breaking Changes
-
 ### Enhancements
-
+* Supports offset based pagination.
```

Comment:
This is already under additions.

Reply:
Isn't this also an enhancement? Or maybe a better question. When does something get added to an enhancement?

Reply:
I think the way we have been doing it is that enhancements are new features on an existing thing and additions are totally new things.